es_ES/cookbook/navigation/passing-data.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: page
+title: "Enviar datos a una nueva pantalla"
+permalink: /cookbook/navigation/passing-data/
+---
+
+A menudo, no solo queremos navegar a una nueva pantalla, sino también pasar algunos datos a la pantalla. Por ejemplo, a menudo queremos pasar información acerca del elemento que hemos pulsado.
+
+Recuerda: las pantallas son Just Widgets&trade;. En este ejemplo, crearemos una 
+lista de pendientes. Cuando se pulsa un pendiente, navegaremos a una nueva pantalla (Widget) que muestra información sobre dicho pendiente.
```

Comment:
¿dejamos sin traducir List of Todos?, los pone en mayúsculas para recalcar que son clases del código.
Si lo hacemos así hay que poner todo en lugar de pendiente.

Sino yo pondría tarea pendiente en lugar de solo pendiente.

Reply:
Cambie "al objeto Todo" ya que es una instancia de la clase.